content/how-to/remediate-alerts/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Remediate alerts"
-description: "How to remediate alerts"
-lead: "How do I remediate secrets in code once they have been found by BluBracket?"
+title: "How to remediate alerts in code once they have been found by BluBracket"
```

Comment:
Long titles are good for FAQs, but shorter titles work better in the usage guide. I wouldn't have changed this one.